client/app/vps/additional-disk/index.js

```diff
@@ -0,0 +1,23 @@
+import component from './vps-additional-disk.component';
+
+import vpsAdditionnalDiskOrderModule from './order';
+
+const moduleName = 'vpsAdditionnalDiskModule';
+
+angular
+  .module(moduleName, [
+    vpsAdditionnalDiskOrderModule,
+  ])
+  .component(component.name, component)
+  .config(/* @ngInject */($stateProvider) => {
```

Comment:
Try to extract the routing in a separate file and 
```suggestion
  import routing from './routing.js'
```

Reply:
```js
import routing from './routing'
```
;)